cog_conversion.py

```diff
@@ -145,7 +145,7 @@ async def togif(self, ctx):
         :param ctx: discord context
         :mediaparam video: A video.
```

Comment:
needs to be updated to reflect changes. `$help` takes from docstrings so it'll work.